### DISCUSSION TITLE
I'd like a service for connecting to a different server

### DISCUSSION THREAD

**Discussion by hmorkemo** [home-assistant/frontend#15461] [Other feature requests] [answered]
I know it's possible to switch between servers with a three-finger swipe.
However, this isn't always as reliable/simple as it should be. Particularly on an iPad.
What would be very helpful is to have some service for doing this.
Then I can create a button in my Home UI switching to my Cabin, and vice versa.

**Comment by frenck** [comment, marked as answer]
This is not a frontend-related discussion, as this is a specific feature of the ios apps. Even if there would be frontend support possible, it would be needed to be exposed by the apps.

You can find the repository of the iOS apps here:

<https://github.com/home-assistant/ios>

Thanks 👍 

../Frenck

**Reply by hmorkemo** [reply]
Thanks! I got lost there for a moment. Going to the right repository now!
